Formula/p/poetry.rb

```diff
@@ -19,22 +19,26 @@ class Poetry < Formula
   end
 
   depends_on "cmake" => :build # for rapidfuzz
-  depends_on "cffi"
-  depends_on "keyring"
-  depends_on "pycparser"
   depends_on "python-certifi"
-  depends_on "python-packaging"
   depends_on "python@3.12"
-  depends_on "virtualenv"
+
+  on_linux do
+    depends_on "python-cryptography"
```

Comment:
Just curious, on macOS what satisfies the py cryptography dep? Since we don't include it as a resource here.

Reply:
nothing it's a linux-only dep from [keyring](https://github.com/jaraco/keyring/blob/bb73eea60825829eb355afc73bdc7c21427d2668/setup.cfg#L22-L23) -> SecretStorage

https://github.com/Homebrew/homebrew-core/blob/ef2cd60da03a7a5ea9be47dc231d36c9a8ca1ae8/Formula/k/keyring.rb#L24-L36